test/ngAnimate/animateSpec.js

```
@@ -292,6 +292,129 @@ describe("animations", function() {
       });
     });
 
+    describe('customFilter()', function() {
+      it('should clear the `customFilter` if no function is passed',
+        module(function($animateProvider) {
+          expect($animateProvider.customFilter()).toBeNull();
+
+          $animateProvider.customFilter(angular.noop);
+          expect($animateProvider.customFilter()).toEqual(jasmine.any(Function));
+
+          $animateProvider.customFilter({});
```

perhaps we should throw instead of silently ignoring?